frontend/app/src/scenes/Home/scenes/AddSubCategory/index.tsx

```diff
@@ -54,25 +64,59 @@ class AddSubCategory extends React.Component<
       );
   };
 
+  fetchSubCategoryDetails = (subCategoryId: string) => {
+    axios
+      .get(
+        window.location.origin +
+          `/core/academix/sub-categories/${subCategoryId}`
+      )
+      .then((result: AxiosResponse<SubCategory>) => {
+        if (result.status == 200) {
+          this.setState({
+            subcategory: result.data,
+            isLoading: false,
+          });
+        } else {
+          throw new Error();
+        }
+      })
+      .catch((error) =>
+        handleApiError(
+          error,
+          'Something went wrong when trying to load the subcategory'
+        )
+      );
+  };
+
   onFinish = (values: any) => {
+    let statusCode: number, method: Method, url;
+    if (!this.state.isComponentEdit) {
+      statusCode = 201;
+      method = 'post';
+      url = '/core/academix/admin/sub-categories/';
+    } else {
+      statusCode = 202;
```

Comment:
This should fix in the backend

Reply:
so what should I do until then?

Reply:
you can fix the backend with this PR. It's a single line change

Reply:
done